Directory.Build.targets

```diff
@@ -105,6 +105,11 @@
     <Compile Include="$(SharedSourceRoot)ReferenceAssemblyInfo.cs" LinkBase="Properties" />
   </ItemGroup>
 
+  <PropertyGroup Condition="'$(Language)' == 'C#' AND '$(IsReferenceAssemblyProject)' == 'true'">
+    <!-- Reference assemblies should always use Major.Minor.0.0 for assembly versions even during servicing. Only the package version should be updated -->
+    <AssemblyVersion>$(AspNetCoreMajorVersion).$(AspNetCoreMinorVersion).0.0</AssemblyVersion>
```

Comment:
This logic is fine so long as everything you are shipping here is only supporting .NETCore.  If you are supporting .NETFramework (desktop) with any of these reference assemblies then they need to match the implementation assembly version for binding redirects to work correctly.  This may be irrelevant if you are only building these for ref/runtime packs on .NETCore.